egs/ru_open_stt/asr1/RESULTS.md

```diff
@@ -0,0 +1,24 @@
+# Transformer (elayers=12, dlayers=6, units=2048, 4 GPUs, specaug)
```

Comment:
How about keeping some of the old results to see the difference?

Reply:
The new results are not directly comparable to the old results because they use different subsets for testing. Current release of the dataset added special validation subsets with manually checked annotations and they can be used for comparisons in the future.